CMakeLists.txt

```
@@ -521,26 +521,33 @@ if(CIRCT_SLANG_FRONTEND_ENABLED)
   if(CIRCT_SLANG_BUILD_FROM_SOURCE)
     # Build slang as part of CIRCT (see https://sv-lang.com/building.html)
     message(STATUS "Building slang from source")
-    set(original_CXX_FLAGS ${CMAKE_CXX_FLAGS})
+    set(original_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
+    set(original_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
     set(CMAKE_CXX_FLAGS "")
+    set(BUILD_SHARED_LIBS OFF)  # statically link slang into ImportVerilog
     include(FetchContent)
     FetchContent_Declare(
       slang
       GIT_REPOSITORY https://github.com/MikePopoloski/slang.git
-      GIT_TAG v2.0
+      GIT_TAG v3.0
       GIT_SHALLOW ON)
+    set(SLANG_INCLUDE_TOOLS OFF CACHE INTERNAL "")
+    set(SLANG_INCLUDE_TESTS OFF CACHE INTERNAL "")
+    set(SLANG_INCLUDE_INSTALL OFF CACHE INTERNAL "")
+    set(SLANG_USE_MIMALLOC OFF CACHE INTERNAL "")
```

```suggestion
    set(SLANG_USE_MIMALLOC OFF CACHE INTERNAL "")
    set(FETCHCONTENT_TRY_FIND_PACKAGE_MODE "NEVER")
```
I tried this in my machine and works well